src/sbvr-api/sbvr-utils.coffee

```diff
@@ -540,14 +541,34 @@ exports.runRule = do ->
 		.nodeify(callback)
 
 exports.PinejsClient =
-	class PinejsClient extends PinejsClientCore(_, Promise)
-		_request: ({ method, url, body, tx, req, custom }) ->
+	class PinejsClient extends PinejsClientCore(_, Promise, mimemessage)
+		compileBatch: (data) -> _.map(data, batchify)
+		_request: ({ method, url, body, multipart, tx, req, custom }) ->
+			if _.isArray(multipart?.data)
```

Comment:
If `multipart.data` exists what can it be other than an array? And if it is something else should we return an error instead?